src/Core/Search/SearchParameters.php

```diff
@@ -58,10 +50,13 @@ public function __construct(AdminFilterRepository $adminFilterRepository)
     public function getFiltersFromRequest(Request $request, $filterClass)
     {
         $filters = [];
-        $defaultValues = $filterClass::getDefaults();
 
-        foreach (self::FILTER_TYPES as $type) {
-            $filters[$type] = $request->get($type, $defaultValues[$type]);
+        foreach (static::FILTER_TYPES as $type) {
```

Comment:
Can you keep the `self` other someone won't be able to implement and add a type in FILTER_TYPES

Reply:
FILTER_TYPES is a constant

Reply:
ok I switched with `self` and squashed the branche, if it's ok for you it can go to QA